setup.py

```diff
@@ -52,16 +52,16 @@ def get_version(*file_paths):
     classifiers=[
         'Development Status :: 3 - Alpha',
         'Framework :: Django',
-        'Framework :: Django :: 1.8',
-        'Framework :: Django :: 1.9',
-        'Framework :: Django :: 1.10',
+        'Framework :: Django :: 2.0',
+        'Framework :: Django :: 2.1',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)',
         'Natural Language :: English',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
     ],
     extras_require={
-        'Django': ['Django>=1.8,<1.11'],
+        'Django': ['Django>=2.0,<2.2'],
```

Comment:
Shouldn't this range allow 1.11?  Also, the Django 1.11 trove classifier is still missing above.

Reply:
Ouch, thanks for catching this, my bad.